crates/internal/src/main.rs

```diff
@@ -104,6 +104,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                         spark_wallet::WalletEvent::StreamDisconnected => println!("Disconnected from Spark server."),
                         spark_wallet::WalletEvent::Synced => println!("Synced"),
                         spark_wallet::WalletEvent::TransferClaimed(transfer) => println!("Transfer claimed: {}", transfer.id),
+                        spark_wallet::WalletEvent::FoundClaimableTransfer(transfer) => println!("Found claimable transfer: {}", transfer.id),
```

Comment:
We have TransferedClaimed, perhaps we can rename that to TransferClaiming or TransferClaimStarted ?

Reply:
Renamed to `TransferClaimStarting`